csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -70,8 +71,19 @@ async def init(command):
         os.makedirs(temp_path, exist_ok = True)
         # Generate Tar file for Logs Folder.
         tar_file_name = os.path.join(temp_path, f"{component_name}_{bundle_id}.tar.gz")
-        if all(map(os.path.exists, component_data[component_name])):
-            Tar(tar_file_name).dump(component_data[component_name])
+        files_to_compress = list()
+        for file_path_pattern in component_data[component_name]:
+            # Check all the files matching naming pattern and returns list of them.
+            matching_files = glob.glob(file_path_pattern)
+            if matching_files:
+                files_to_compress.extend(matching_files)
+            elif os.path.exists(file_path_pattern):
```

Comment:
This will be covered by if condition?

Reply:
glob only matched the pattern if a file_path doesn't have a pattern it returns [] even when file exists.
So the second condition